conf/application.conf

```diff
@@ -145,7 +147,7 @@ metrics {
     durationUnit = SECONDS
     showSamples = true
     jvm = true
-    enabled = false
+    enabled = true
```

Comment:
What is this change for?

Reply:
Without enabled metrics locally you're not able to use this inside the tests, it's failing during creating controllers that use this metrics.

To be honest I'm not sure why, but all our services has this enabled locally.